src/createReducer.test.ts

```diff
@@ -47,4 +46,57 @@ test('createReducer should create a redux reducer that updates the state on upda
     expect(newState.d).not.toBe(defaultState.d);
     expect(newState.d.e).toBe(3);
     expect(defaultState.d.e).toBe(2);
+});
+
+test('createReducer should create a redux reducer and an UPDATE_ACTION should update the state and a RESET_ACTION reset to default', () => {
```

Comment:
Het updaten van de state wordt al getest in de vorige test. Je zou hier gewoon alleen de reset kunnen testen.